### PR TITLE
feat: add nord theme, update slider screenshots

### PR DESCRIPTION
**Pull request by Tetrax-10**
@theRealPadster can you change this image to Nord theme. becuase that image looks like a default spotify.
image

also add nord spotify here
image

### CODE REVIEW

File: docs/advanced-usage/themes.md [hunk 1]
```diff
@@ -13,4 +13,4 @@ For this reason, the theming heroes of the Spicetify community have created a hu
 2. [NYRI4/Comfy-spicetify](https://github.com/NYRI4/Comfy-spicetify)
 3. [williamckha/spicetify-fluent](https://github.com/williamckha/spicetify-fluent)
 4. [JulienMaille/dribbblish-dynamic-theme](https://github.com/JulienMaille/dribbblish-dynamic-theme)
-5. Insert your theme here!
+5. [Tetrax-10/Nord-Spotify](https://github.com/Tetrax-10/Nord-Spotify)
```
Comment (theRealPadster) [inline docs/advanced-usage/themes.md:16]: Can you add the "add your theme here" entry as number 6? We don't want to remove it.

**Comment by theRealPadster** [comment]
I believe that first screenshot is an old version of Comfy? It's got the gradient play bar. But feel free to change it, or add a screenshot or two from Nord into the carousel. The files are all in this repo, so you can do in this same PR

**Review by theRealPadster — CHANGES_REQUESTED** [review]

**Comment by Tetrax-10** [comment]
> 

i know all those images are there in the repo but i dont know to take a ss with that rounded corner with faded bg. all images should be even so its better you take a ss and commit it

**Comment by theRealPadster** [comment]
> i know all those images are there in the repo but i dont know to take a ss with that rounded corner with faded bg. all images should be even so its better you take a ss and commit it

Oh, right, I forgot I made them that way. That's how macos saves them. I'll see what I can do.

**Comment by theRealPadster** [comment]
Hey @afonsojramos , could you replace the themes screenshot with something that doesn't look so stock? (It looks like your account.) I was going to but my screen isn't big enough to keep the same resolution. 
image

**Review by theRealPadster — APPROVED** [review]
Looks good to me. Feel free to update that themes image, but can merge as far as I'm concerned.

**Comment by Tetrax-10** [comment]
Looks good to me too.

> Hey @afonsojramos , could you replace the themes screenshot with something that doesn't look so stock? (It looks like your account.) I was going to but my screen isn't big enough to keep the same resolution. image

I Recommend Bloom/ziro/sleek

**Comment by afonsojramos** [comment]
@theRealPadster It's fine if it isn't the same res! Actually it would be better if it was a smaller res!

**Comment by theRealPadster** [comment]
> @theRealPadster It's fine if it isn't the same res! Actually it would be better if it was a smaller res!

Well I just meant so those three in that block are the same size. Should we just redo them all then?

**Comment by theRealPadster** [comment]
I don't feel like redoing all those screenshots right now, so I'm gonna merge this in, and you can make another issue/PR about updating them if it really matters.

**Pull request merged**